src/controllers/UsersController.php

```diff
@@ -386,6 +386,42 @@ public function deleteUser($request, $db)
         $view->setResponseCode(204);
     }
 
+    /**
+     * Allow users to be set as trusted
+     *
+     * @param $request
+     * @param $db
+     *
+     * @return null
+     *
+     * @throws Exception
+     *
+     */
+    public function setTrusted($request, $db)
+    {
+        if (false == ($request->getUserId())) {
+            throw new Exception("You must be logged in to change a user account", 401);
+        }
+
+        $user_mapper = $this->getUserMapper($db, $request);
+        if (!$user_mapper->isSiteAdmin($request->getUserId())) {
+            throw new Exception("You must be an admin to change a user's trusted state", 401);
```

Comment:
As the user already *is* authorized (otherwise she would have been thrown out a few lines above) the action of adding the trusted-flag is not possible for the currently logged in user. I see 401 as "authentication-issue" whereas 403 is an "authorization-issue" (like http://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses)

But I don't mind either. I'd connect this to https://joindin.jira.com/browse/JOINDIN-709 and https://joindin.jira.com/browse/JOINDIN-582 and leave it as it is ;)